apps/vmq_events_sidecar/test/vmq_events_sidecar_SUITE.erl

```diff
@@ -84,15 +86,15 @@ on_register_empty_properties_test(_) ->
 on_publish_test(_) ->
     enable_hook(on_publish),
     Self = pid_to_bin(self()),
-    [ok] = vmq_plugin:all(on_publish,
+    [ok,ok] = vmq_plugin:all(on_publish,
```

Comment:
If we use all fn then does it mean that if the first hook return next/ok/error, the next hook in the chain will also be called?

Reply:
There are places where all is used and some places all_till_ok and at the same time some hooks return ok upon success and some return next. Can we identify and keep all the hooks and their return values in a standard fashion based on some logic/reason?